### DISCUSSION TITLE
Partitions are missing in Topic after perforiming Backup and Restore.

### DISCUSSION THREAD

**Discussion by syedval** [strimzi/strimzi-kafka-operator#9738] [Q&A]
Hi All,

while performing Backup and Restore. we are able to see the topics being restored but somehow partitions are missing or incorrect which are present before Backup.

we have performed below steps to reproduce the issue:
1) Create Kafka Operand with backup & restore enabled and create some topics.

2) Take zookeeper backup

3) Delete operand and PVCs

4) Create operand again, restore backup.

5) Topics created earlier are not restored. if restored all partitions are not visible immediately.

could someone help us how can we resolve this issue.

Thanks,
Syed.

**Comment by scholzj** [comment]
Strimzi does not have any direct backup and restore support. And doing backup and restore in Kafka is fairly complicated. So for a start, you would need to explain what exactly you understand under it and also what exactly is the issue you see.

**Reply by scholzj** [reply]
For the record, Kafka in general does not have any real backup / restore support other than mirroring. The main point is that you have to explain what you consider backup / restore and not consider it to be somehow defined.
